Content.Client/Lobby/UI/CharacterSetupGuiSavePanel.xaml.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.Lobby.UI
+{
+    [GenerateTypedNameReferences]
+    public sealed partial class CharacterSetupGuiSavePanel : DefaultWindow
+    {
+        public CharacterSetupGuiSavePanel()
+        {
+            RobustXamlLoader.Load(this);
+
+            CancelButton.OnPressed += _ =>
+            {
+                Close();
+            };
+
+            CloseButton.Visible = false;
+        }
+    }
+}
```

Comment:
```suggestion
namespace Content.Client.Lobby.UI;

[GenerateTypedNameReferences]
public sealed partial class CharacterSetupGuiSavePanel : DefaultWindow
{
    public CharacterSetupGuiSavePanel()
    {
        RobustXamlLoader.Load(this);

        CancelButton.OnPressed += _ =>
        {
            Close();
        };

        CloseButton.Visible = false;
    }
}
```

file-scoped namespaces please